### DISCUSSION TITLE
[질문] board 조회수 증가 로직 질문

### DISCUSSION THREAD

**Discussion by kang20** [Kumoh-talk/kumoh-talk-Backend#59] [Q&A]
![image](https://github.com/user-attachments/assets/bc3e6e3d-7f51-413f-84d8-f5790b475c89)

현재는 조회수 증가 로직이 조회하면 view 테이블 수를 1 증가하는 식으로 구성하고 있습니다. 

user를 통해 조회수 증가 제어는 게시물 조회가 Annonymous 도 가능하기 때문에 불가능한데

다른 의견 있으신가요 아니면 그냥 이대로 진행할까요?

**Comment by siwan9** [comment]
저번 회의 때 클라이언트 IP주소를 통해 몇 시간 동안은 동일 IP주소의 게시물 중복 조회를 허용하지 않는다고 회의하긴 했었습니다

**Reply by kimgunwooo** [reply]
이게 제일 나을 것 같긴하네요. 이거랑 디도스 같은 공격 대비해서 처리율 제한 기능이 있으면 좋을 것 같긴한데, mvc 기반이라 인터셉터나 필터로 만들어야 될 것 같기도하고..

구현한다면 해당 메서드에 대해서만 IP 조회해서 허용하지 않는방법으로 할지,
아님 처리율 제한 기능을 따로 구현하는게 좋을지 얘기가 필요할 것 같긴하네요

**Reply by kang20** [reply]
일단 그럼 보류하도록 하겠습니다
